charts/zenko/values.yaml

```diff
@@ -95,4 +95,25 @@ redis-ha:
     create: true
   replicas:
     servers: *nodeCount
-    sentinels: *nodeCount
\ No newline at end of file
+    sentinels: *nodeCount
+
+grafana:
+  enabled: false
+  # When 'grafana.enabled' is true, a grafana chart is deployed as part of the Zenko stack
+  ingress:
+    enabled: true
```

Comment:
No. Don't enable `Ingress`es by default, since there's no way to know the host to use etc.